src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp/Functions/RefreshSearchableDocument/RefreshSearchableDocumentFunction.cs

```diff
@@ -8,26 +8,35 @@ public class RefreshSearchableDocumentFunction(ISearchableDocumentCreator search
 {
     [Function(nameof(RefreshSearchableDocument))]
     [QueueOutput("%SearchableDocumentCreatedQueueName%")]
-    public async Task<SearchableDocumentCreatedMessageDto> RefreshSearchableDocument(
+    public async Task<SearchableDocumentCreatedMessageDto[]> RefreshSearchableDocument(
         [QueueTrigger("%RefreshSearchableDocumentQueueName%")]
         RefreshSearchableDocumentMessageDto message,
         FunctionContext context)
     {
+        if (string.IsNullOrEmpty(message.PublicationSlug))
+            return [];
```

Comment:
We are tending to use braces everywhere. Using braces here would be more in keeping with the project's existing code style.